src/HttpUpgradeProtocolAccess.cc

```diff
@@ -53,7 +51,7 @@ HttpUpgradeProtocolAccess::configureGuard(ConfigParser &parser)
     if (!rawProto)
         throw TextException(ToSBuf("expected a protocol name or ", ProtoOther), Here());
```

Comment:
This should fix the build:

```suggestion
        throw TextException(ToSBuf("expected a protocol name or ", ProtoOther()), Here());
```